documentation/architecture/openspec/project.md

```diff
@@ -1,31 +1,68 @@
 # Project Context
 
 ## Purpose
-[Describe your project's purpose and goals]
+Mimeogram is a command-line tool for **exchanging collections of files with Large Language Models (LLMs)**. It bundles multiple files into a single clipboard-ready document while preserving directory structure and metadata, making it ideal for code reviews, project sharing, and LLM interactions. It supports interactive reviews, clipboard integration, and path protection.
```

Comment:
"This is a" rather than "Mimeogram is a". If we do not refer to the project/package name, it makes this document more maintainable if we need to rename the project later.